shacl12-core/index.html

```diff
@@ -1546,7 +1538,8 @@ <h3>Property Shapes</h3>
 					<span data-syntax-rule="PropertyShape">A <dfn data-lt="property shapes">property shape</dfn> is a <a>shape</a> in the <a>shapes graph</a>
 					that is the <a>subject</a> of a <a>triple</a> that has <code>sh:path</code> as its <a>predicate</a>.</span>
 					<span data-syntax-rule="path-maxCount">A shape has at most one <a>value</a> for <code>sh:path</code>.</span>
-					<span data-syntax-rule="path-node">Each <a>value</a> of <code>sh:path</code> in a shape must be a <a>well-formed</a> <a>SHACL property path</a>.</span>
+					<span data-syntax-rule="path-node">Each <a>value</a> of <code>sh:path</code> in a shape must be a <a>well-formed</a> <a>SHACL property path</a>
+					or (if it is not a <a>well-formed</a> <a>SHACL property path</a>) a <a>well-formed</a> <a>node expression</a> but not a <a>literal</a>.</span>
```

Comment:
As said, keeping Node Expressions and Property Paths mutually exclusive should simplify this definition